src/main/resources/application.yml

```diff
@@ -142,7 +142,7 @@ initializr:
         - versionRange: "[1.5.0.RELEASE,2.0.0.M1)"
           version: 1.2.51
         - versionRange: "[2.0.0.M1,2.1.0.M1)"
-          version: 1.2.71  
+          version: 1.2.71
```

Comment:
That trailing whitespace seems unrelated. Can you please fix it?

Reply:
Sure...my editor automatically removes trailing whitespace at the end of lines, which is why that is there. But I can probably put it back.

Reply:
Oh geez...there were 2 spaces and I only put back 1.

Reply:
Okay...fixed now.

Reply:
@snicoll Is the change I've provided good for you? Anxious to get this merged to address a customer need.